doc/operation-and-maintenance/Rolling-upgrade.md

```diff
@@ -25,30 +25,35 @@ A healthy cluster should list all nodes that are part of the cluster.
 
 Should you have any issues related to node clustering, please refer to [Cluster configuration and node management](Cluster-configuration-and-node-management.md) section.
 
-### 2. Apply the changes from the migration guide.
+### 2. Copy the configuration file.
+
+Make a copy of the configuration file before the upgrade, as some package managers might override your custom configuration with the default one.
+Please note that since version 4.1 `*.cfg` MongooseIM configuration format is no longer supported and needs to be rewritten to the new `*.toml` format.
```

Comment:
Minor, might be subjective: "rewrite to" sounds strange to me... should it be "rewrite **in** the new format"?